app/assets/javascripts/actions/conference_actions.es6.jsx

```diff
@@ -86,6 +74,14 @@
       };
     }
 
+    storeListAttribute(key, value, index) {
```

Comment:
Seems like a not unreasonable idea, but it'll require a check for an index in storeAttribute, however most storeAttributes don't require any index, and it'll make this store's `StoreAttribute` different from all other storeAttributes. I'd rather just leave it as is, but your call.